server/src/main/java/org/jboss/as/server/deployment/module/VFSResourceLoader.java

```diff
@@ -123,18 +141,41 @@ public Manifest run() throws IOException {
                 throw new UndeclaredThrowableException(e);
             }
         }
+        // "A multi-release jar file is a jar file that contains a manifest with a main attribute named "Multi-Release", [...]"
+        multiRelease = manifest != null && manifest.getMainAttributes().containsKey(MULTI_RELEASE_NAME);
```

Comment:
Hi @dmlloyd, apparently the value of that attribute must be `true` for the multi-release jar to be considered enabled https://mail.openjdk.java.net/pipermail/core-libs-dev/2019-January/058131.html

Reply:
Thanks for the note.  Could you please open a JIRA (it can have "minor" priority)?  And a PR if you want to.  Thanks.

Reply:
Done - https://issues.jboss.org/browse/WFCORE-4298

Reply:
Thanks Jaikiran!